src/hal/stm32f7/init.rs

```diff
@@ -0,0 +1,348 @@
+// Zinc, the bare metal stack for rust.
+// Copyright 2014 Vladimir "farcaller" Pouzanov <farcaller@gmail.com>
```

Comment:
Yep, I realised it later, sorry for the noise (came to be my go-to comment on these files).